### ISSUE TITLE
/playanimation cannot find animation

### ISSUE THREAD

**Issue by Xterionix** [Blockception/BC-Minecraft-Bedrock-Command#96]
It says the animation cannot be found but I have it in the RP
![image](https://github.com/Blockception/BC-Minecraft-Bedrock-Command/assets/72647213/9d04b005-e65c-45eb-9498-fb2d65553ffe)
![image](https://github.com/Blockception/BC-Minecraft-Bedrock-Command/assets/72647213/9e65c6b5-187e-49f4-882d-1693455871c3)

**Comment by DaanV2** [comment]
hmm, the fact that that code lens is there means it should work, does it presistent on reload, or on resave of both files?

**Comment by Xterionix** [comment]
Yeah I've saved both files and tried it again but it still shows the same error

**Comment by DaanV2** [comment]
Wait shouldnt it be the name of the animation in you client file and not the animation file 🤔

**Comment by Xterionix** [comment]
That is what that is

**Comment by Xterionix** [comment]
The file itself is called menefeeceratops.animation.jons

**Comment by DaanV2** [comment]
Thats behavior I didnt know worked 🤔

**Comment by Xterionix** [comment]
/playanimation can play any anim regardless of if it is listed in the client entity file or not

**Comment by DaanV2** [comment]
Will be alot more changes, but ill fix it
